profiles/base.yaml

```diff
@@ -251,6 +251,7 @@ appraisal_training:
 full_pouch_container:
 classes_to_teach:
 avoid_athletics_in_justice: false
+athletics_outdoorsmanship_rooms:
```

Comment:
If this is an array, it needs an entry in `base-empty.yaml`

Reply:
Done. What's this actually do?

Reply:
If someone overrides a setting in their YAML such as `athletics_outdoorsmanship_rooms:` then `get_settings.athletics_outdoorsmanship_rooms` will return `nil`, not an array. Sometimes that's ok, but sometimes that causes an error (e.g. if you're using an Array-specific function in your script).

Base-empty is almost solely for settings which are arrays and hashes. It provides a default value if the setting is `nil` when loaded.

Reply:
Ah, thanks for the explanation.